docs/io/unified.rst

```diff
@@ -821,3 +905,86 @@ To write to a new file, the ID of the table should also be specified (unless
 When writing, the ``compression=True`` argument can be used to force
 compression of the data on disk, and the ``overwrite=True`` argument can be
 used to overwrite an existing file.
+
+.. _table_serialization_methods:
+
+Table serialization methods
+===========================
+
+Astropy supports fine-grained control of the way to write out (serialize)
+the columns in a Table.  For instance if you are writing an ISO format
+Time column to an ECSV ASCII table file, you may want to write this as a pair
+of JD1 / JD2 floating point values for full resolution (perfect round-trip), or
+as a formatted ISO date string so that the values are easily readable by you or
+other applications.
+
+The default method for serialization depends on the format (FITS, ECSV, HDF5).
+For instance HDF5 is a binary format and so it would make sense to store a Time
+object as JD1 / JD2, while ECSV is a flat ASCII format and commonly you
+would want to see the date in the same format as the Time object.  The defaults
+also reflect an attempt to minimize compatibility issues between astropy
+versions.  For instance it was possible to write Time columns to ECSV as
+formatted strings in a version prior to the ability to write as JD1 / JD2
+pairs, so the current default for ECSV is to write as formatted strings.
+
+The two classes which have configurable serialization method are
+`~astropy.time.Time` and `~astropy.table.MaskedColumn`.  See the sections
+on Time `Details`_ and `Masked columns`_, respectively, for additional
+information.  The defaults for each format are listed below:
+
+====== ==================== ===============
+Format    Time                MaskedColumn
+====== ==================== ===============
+FITS    ``jd1_jd2``          ``null_value``
+ECSV    ``formatted_value``  ``null_value``
+HDF5    ``jd1_jd2``          ``data_mask``
+YAML    ``jd2_jd2``            ---
+====== ==================== ===============
+
+As an example, start by making a table with a Time column and masked column:
+
+  >>> import sys
+  >>> from astropy.time import Time
+  >>> from astropy.table import Table, MaskedColumn
+
+  >>> t = Table(masked=True)
+  >>> t['tm'] = Time(['2000-01-01', '2000-01-02'])
+  >>> t['mc1'] = MaskedColumn([1.0, 2.0], mask=[True, False])
+  >>> t['mc2'] = MaskedColumn([3.0, 4.0], mask=[False, True])
+  >>> t
+  <Table masked=True length=2>
+             tm             mc1     mc2
+           object         float64 float64
+  ----------------------- ------- -------
+  2000-01-01 00:00:00.000      --     3.0
+  2000-01-02 00:00:00.000     2.0      --
+
+Now specify that you want all `~astropy.time.Time` columns written as JD1 / JD2
+and the ``mc1`` column written as a data / mask pair and
+write to ECSV:
+
+.. doctest-skip::
```

Comment:
I'm surprised the doctest fails; normally it happily ignores whitespace differences.